modules/apps/commerce/commerce-currency-web/src/main/resources/META-INF/resources/edit_commerce_currency.jsp

```diff
@@ -53,7 +53,7 @@ if (Validator.isNotNull(backURL)) {
 				<aui:fieldset>
 					<aui:input bean="<%= commerceCurrency %>" model="<%= CommerceCurrency.class %>" name="name" />
 
-					<aui:input bean="<%= commerceCurrency %>" model="<%= CommerceCurrency.class %>" name="code" />
+					<aui:input bean="<%= commerceCurrency %>" helpMessage="currency-code-help" model="<%= CommerceCurrency.class %>" name="code" />
```

Comment:
should be set as `readonly="readonly"` or `readonly="<%= true %>"`, check which one is the proper one for this use case

Reply:
@luca-pellizzon , if we use readonly, user cannot input a value even when adding a currency.

Reply:
Actually, I think I can resolve this. We can do a null check.